Short-Answer/Algorithms_Answers.md

```diff
@@ -14,10 +14,15 @@ c)This one is a O(n) as the size of the input increases, the runtime or space us
 n-story building
 many eggs
 
-#pseudocode
-
 if egg is at floor x or at floor greater then floor x  
  then egg will eqaul to break
 
 or elase if egg at is at floor less then floor x
 then egg will equal to not break
+
+#pseudo code
+
+if eggs are thrown out x floor and break
+then f is eqaul or greater x floor  
+ otherwise if eggs thrown and do not break
+then x floor is less than f
```

Comment:
This is a good start to the pseudocode, but ti's not a complete plan. From this plan, we know if f is greater than or less than the floor we're on, but how can we actually find the exact value of f? What would be the full process to search for and find f?